### DISCUSSION TITLE
select 组件 设置分组时 scrollTo中的index 应该如何使用？

### DISCUSSION THREAD

**Discussion by BluesVN** [ant-design/ant-design#34831] [Q&A]
select 组件的下拉菜单可以自定义滚动到指定下标（使对应option出现在可视区域内）。没有使用分组时下标0代表下拉菜单中第一个元素。
如果使用OptGroup分组，index是什么规律呢，父节点和子节点的index是同一层级吗？

**Comment by BluesVN** [comment]
我测试的结果是，使用OptGroup分组 index没有分层级。无论是父级节点 还是子节点，都按照lit中从上到下一次排列。

**Comment by vaynevayne** [comment]
select 组件的下拉菜单 支持 scrollTo  ?
